privacy-snapshot.json

```diff
@@ -13,6 +13,7 @@
   "authentication.api.cx.metamask.io",
   "bafkreifvhjdf6ve4jfv6qytqtux5nd4nwnelioeiqx5x2ez5yrgrzk7ypi.ipfs.dweb.link",
   "bafybeidxfmwycgzcp4v2togflpqh2gnibuexjy4m4qqwxp7nh3jx5zlh4y.ipfs.dweb.link",
+  "base-mainnet.infura.io",
```

Comment:
As base network is added by default, we needed to update the privacy snapshot to include the infura RPC calls.